### PR TITLE
Updated index.md

### PR DESCRIPTION
**Pull request by luis-cedillo**
I made some slight changes such as including the pronunciation of some terms.

Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following

- [] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/.github/CONTRIBUTING.md).
- [] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

File: src/pages/cplusplus/input_and_output/index.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ title: Input and Output
 
 ## Input and Output
 
-Input stream objects in C++ use 'cin'. This is used to save input data into the program.
+Input stream objects in C++ use 'cin'. This is pronounced "see in". This is used to input user data into the program.
```
Comment (jonathan-grah) [inline src/pages/cplusplus/input_and_output/index.md]: ~~input user data into the program~~ probably should be *input data into the program* as the term "user" isn't broad enough.

Comment (luis-cedillo) [reply src/pages/cplusplus/input_and_output/index.md]: Sounds good.

**Comment by amandeepmittal** [comment]
@luis-cedillo Please update the commit with changes requested.

**Comment by domwakeling** [comment]
Marked as stale and closed due to inactivity. :disappointed:

Please make appropriate changes if you have the time

**Pull request closed without merging**